src/app/common/utility/isBuyNowCart.ts

```diff
@@ -0,0 +1,3 @@
+export default function isBuyNowCart(): boolean {
+    return window.location.pathname.split('/').pop()?.length === 36;
```

Comment:
Do we need to check for length 36? Can we check the inverse if the value is not `checkout` or `embedded-checkout` in that case